ddtrace/opentracer/propagation/http.py

```diff
@@ -0,0 +1,79 @@
+import logging
+
+from opentracing import SpanContextCorruptedException
+from ddtrace.propagation.http import HTTPPropagator as DDHTTPPropagator
+from ddtrace.propagation.http import HTTP_HEADER_TRACE_ID, HTTP_HEADER_PARENT_ID, HTTP_HEADER_SAMPLING_PRIORITY
+
+from ..span_context import SpanContext
+
+from .propagator import Propagator
+
+
+log = logging.getLogger(__name__)
+
+HTTP_BAGGAGE_PREFIX = 'ot-baggage-'
+HTTP_BAGGAGE_PREFIX_LEN = len(HTTP_BAGGAGE_PREFIX)
+
+# valid baggage keys to check span validity
+VALID_BAGGAGE_KEYS = {
+    HTTP_HEADER_TRACE_ID: True,
+    HTTP_HEADER_PARENT_ID: True,
+    HTTP_HEADER_SAMPLING_PRIORITY: True,
+}
+
+
+class HTTPPropagator(Propagator):
+    """OpenTracing compatible HTTP_HEADER and TEXT_MAP format propagator.
+
+    `HTTPPropagator` provides compatibility by using existing OpenTracing
+    compatible methods from the ddtracer along with new logic supporting the
+    outstanding OpenTracing-defined functionality.
+    """
+
+    slots = ['_dd_propagator']
```

Comment:
I think you want [`__slots__`](https://docs.python.org/2.7/reference/datamodel.html#slots) instead of `slots` here?